documentation/man/features/deploy.rst

```diff
@@ -65,6 +66,9 @@ OPTIONS
 -r, \--reboot:
   Reboot machine after deploy.
 
+\--no-reboot:
+  Do not reboot machine after deploy.
```

Comment:
Maybe:
Do not reboot the machine after deploy even if the reboot option is set in the config file.